Android/app/src/main/java/SampleMOdel.java

```diff
@@ -0,0 +1,53 @@
+import com.google.android.gms.maps.model.LatLng;
```

Comment:
What is the SampleModel needed for ? It just has LatLng ? Why do we need it
@rakheegupta